### DISCUSSION TITLE
110 Yards is back online 🎉

### DISCUSSION THREAD

**Discussion by mdryden** [110yards/fantasy#237] [Announcements]
[Previous discussion](https://github.com/mdryden/110yards/discussions/234)

RIP the CFL's API.  Long live the new 110 yards API!

I have managed to cobble together some stats which should work for this season, and as a result, we are back online.

Results have been calculated for this week and waivers are now active.  ~~Waivers will NOT run tomorrow as they normally would, they will run Thursday morning in order to avoid and unfairness because they started so late.~~ I messed this up, sorry.  Waivers ran this morning.  As they were run early and unlocked for several hours, I don't feel that it would be fair to re-enable them.  Sorry for the error and the short waiver period.

Unfortunately with the new data source comes new limitations on scoring.  **The following stat categories are no longer available and therefore will not count for or against any players**:

- Offensive fumbles
- Special teams tackles
- Rogues
- 2 point conversions
- Missed field goal return yardage/touchdowns...maybe - it's unclear.  These might get counted as kick returns now.

As part of the patch all players received new ids.  I have done as much as possible to fix up rosters so the change over is fairly seamless, however, **there are 4 rostered players who are not available on the new roster list** (possibly because they are suspended or not active for some other reason):
- Antonio Williams    (FA)
- Brad Muhammad    (FA)
- DeVontres Dukes    (CGY)
- Emmanuel Rugamba    (BC)
- Jester Weah    (FA)
- JoJo Ward    (FA)
- Jonathan Garibay    (FA)
- Kenny Lawler    (WPG)
- Kwaku Boateng    (FA)
- Marc Liegghio    (FA)
- Matthew Thomas    (FA)
- Mike Glass Iii    (FA)
- Papi White    (FA)
- Patrick Levels    (FA)
- Richard Sindani    (HAM)
- Shawn Lemon    (FA)
- Tobi Antigha    (FA)
- Tyrone Hill Jr    (FA)

 **If you own one of these players, the players has been marked as Free Agents** on your roster.  You should drop the player, as the player on your roster will never accumulate stats. These players will have scored zero this week so there is no impact to scoring.

Commissioners I would encourage you to make a ruling within your league allowing these managers the first right to select these players if/when the appear in the player list in the future.

Finally, the **injury status** feature is temporarily broken.  I hope to have this feature fixed tomorrow.

## Next steps

I'm hoping to fill in some of the scoring gaps in the next few weeks - if I'm able to, please note that I will be re-calculating scores for past weeks to include the missing stats.

Please be patient over the next few days as I do expect there will be some bugs with this new data source.

**Comment by kamikazecanuck** [comment]
Thanks again Pudds.  The site is in a workable state again as far as I'm concerned.  The only big issue is that it thinks Toronto is on a bye but Montreal is.

I'm hoping the Kenny Lawler and Keon Hatcher issues resolve when they are put back on active lists by the league.

**Comment by AdamKonefal** [comment]
FYI DeVontres Dukes is now Tre Odoms-Dukes https://www.cfl.ca/players/tre-odoms-dukes/166753/

His CFL Central ID matches his old one I have stored.

**Comment by kamikazecanuck** [comment]
_pssst..... it actually a "rouge"_.  And it's the most important statistic in Canadian Football!!
